lib/framework.js

```diff
@@ -187,11 +187,13 @@ class Framework {
 			});
 
 			if (!configs || configs.length === 0) {
-				throw new Error("Could not parse ui5.yaml");
+				karma.log("error", ["Could not parse ui5.yaml. Please make sure that the ui5.yaml has a valid format. For reference check: https://github.com/SAP/ui5-project/blob/master/docs/Configuration.md#configuration"]);
```

Comment:
`karma` isn't available as a variable here.
We need to inject the `logger` and create a new logger for our framework.
`const log = logger.create('framework.ui5');`
You can take a look here how other plugins are doing so:
https://github.com/mattlewis92/karma-coverage-istanbul-reporter/blob/ec01a4cacccd1eff9bc1dfcf322a98a825720de0/src/reporter.js#L19-L31

Reply:
Thanks. Is fixed in 742990e